Create Getting Started page for OData client (#100)
MicrosoftDocs/OData-docs: Odata-docs/client/v6/getting-started.md

[DataServiceQuery&lt;Person&gt;](https://docs.microsoft.com/en-us/dotnet/api/microsoft.odata.client.dataservicequery-1).
The `DataServiceQuery<TElement>` class allows you to execute LINQ-enabled queries against
a particular endpoint of the service. `context.People` executes queries that return a
collection of `Person` instances.
The `Person` class is in turn generated based on the `Person` entity in the
OData model and contains the properties defined in the data model.
The generated code also includes enums, and methods corresponding to bound and unbound functions and actions.

Now let's call the `ListPeople()` method from the `Main` method by adding the following
statement in `Main`:
```c#
ListPeople().Wait();
```
Don't forget to add the following using statements as well:
```c#
using System.Collections.Generic;
using System.Threading.Tasks;
```

Now the entire program should look like:
```c#
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Microsoft.OData.SampleService.Models.TripPin;

namespace ODataClientExample
{
    class Program
    {
        static void Main(string[] args)
        {

>             ListPeople().Wait();

Reviewer: You can switch main to async so as to use await or leave a comment for why you are using Wait so that new people to the language don't pick this up and run with it.

---
title: "Basic CRUD Operations"
description: ""

author: habbes
ms.author: habbes
ms.date: 1/27/2019
ms.topic: article
 
---
# Basic CRUD operations

**Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v6.md)]

# Getting Started

> The OData Client library allows you to consume data from and interact with OData services from your .Net applications. There are multiple layers of abstraction available when working with the library: You can use the library directly to interact with any service, or you can use a code generation tool to generate strongly-typed client code that's specfic to the service you are interested in. In this article we are going to focus on the latter approach.

Reviewer 2: ```suggestion
The OData Client library allows you to consume data from and interact with OData services from .Net apps. There are multiple layers of abstraction available when working with the library: 

* You can use the library directly to interact with any service.
* You can use a code generation tool to generate strongly-typed client code that's specific to the service you are interested in.

In this article we are going to focus on the latter approach.
```

> ## Using the OData Connected Service.

Reviewer 2: ```suggestion
## Using the OData Connected Service
```
